docs/samples/browser-plugin-meetings/app.js

```diff
@@ -73,6 +74,39 @@ function changeAuthType() {
   }
 }
 
+function generateWebexConfig({credentials}) {
```

Comment:
created this function, because config was duplicated in 2 places. Apart from this change and the fact that we read the checkbox value for `enableTurnDiscovery` config entry, the rest of the changes in this file are caused by automatic formatting of the code.